app/pb_kits/playbook/pb_home_address_street/_home_address_street.jsx

```diff
@@ -15,6 +15,24 @@ const dot = (houseStyle) =>
   }
 }
 
+function titleize(sentence) {
+    if(!sentence.split) return sentence;
+    var _titleizeWord = function(string) {
+            return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
+        },
+        result = [];
+    sentence.split(" ").forEach(function(w) {
+        result.push(_titleizeWord(w));
+    });
+    return result.join(" ");
+}
```

Comment:
Utilities like this could be very useful across many future kits. What do you think about maybe moving this to a new file under `/app/pb_kits/playbook/utilities/text.js` or similar?

Reply:
Moved to a new file 'utilities/text.js'. Thanks for the help on that!